lib/resources/package.rb

```diff
@@ -45,36 +47,32 @@ def initialize(package_name = nil, opts = {}) # rubocop:disable Metrics/AbcSize
       elsif ['hpux'].include?(os[:family])
         @pkgman = HpuxPkg.new(inspec)
       else
-        return skip_resource 'The `package` resource is not supported on your OS yet.'
+        raise Inspec::Exceptions::ResourceSkipped, 'The `package` resource is not supported on your OS yet.'
       end
 
       evaluate_missing_requirements
     end
 
     # returns true if the package is installed
     def installed?(_provider = nil, _version = nil)
-      return false if info.nil?
       info[:installed] == true
     end
 
     # returns true it the package is held (if the OS supports it)
     def held?(_provider = nil, _version = nil)
-      return false if info.nil?
       return false unless info.key?(:held)
```

Comment:
I think we can remove this too and just it fall through to the boolean comparison, yes?

Reply:
Yup, I see we set it to a bool farther down. Good catch!